modules/analysis-common/src/test/java/org/elasticsearch/analysis/common/CommonAnalysisFactoryTests.java

```diff
@@ -48,7 +48,7 @@ public CommonAnalysisFactoryTests() {
         tokenizers.put("edgengram", EdgeNGramTokenizerFactory.class);
         tokenizers.put("classic", ClassicTokenizerFactory.class);
         tokenizers.put("letter", LetterTokenizerFactory.class);
-        tokenizers.put("lowercase", LowerCaseTokenizerFactory.class);
+        // tokenizers.put("lowercase", XLowerCaseTokenizerFactory.class);
```

Comment:
Why is it commented ?

Reply:
The tests here are explicitly checking that we can load lucene analysis classes.  `LowercaseTokenizer` isn't there any more, so this needs to be removed - the commenting out was just to get tests to pass.